OrbitQt/orbitmainwindow.cpp

```diff
@@ -81,8 +81,17 @@ OrbitMainWindow::OrbitMainWindow(QApplication* a_App,
     setWindowTitle({});
   });
 
+  constexpr const char* kFinalizingCaptureMessage =
+      "<div align=\"left\">"
+      "Please wait while the capture is being finalized..."
+      "<ul>"
+      "<li>Waiting for the remaining capture data</li>"
+      "<li>Processing callstacks</li>"
+      "<li>Cleaning up dynamic instrumentation trampolines</li>"
```

Comment:
Maybe remove the word "trampolines"? Seems too much of a detail that users don't have to know about necessarily.

Reply:
I'm totally fine with it. So just "Cleaning up dynamic instrumentation"?

Reply:
SGTM